lib/actuators/src/ContServo.cpp

```diff
@@ -3,6 +3,7 @@
 ContServo::ContServo(PinName pin) : Servo(pin) {
   m_max_speed = 0;
   m_speed     = 0;
+  m_inverted  = false;
```

Comment:
whats this idea of inverted for? was the previous code broken?

Reply:
This was copied over from that extra ServoMotor we had before, although I don't think it was actually used. Should I just  remove it?

Reply:
im not sure i havent looked into the code. its up to you. if it does nothing, remove it, but be very sure of it before removing it

Reply:
it looks like inverted in important leave it